packages/components/src/mobile/gradient/index.native.js

```diff
@@ -97,3 +112,4 @@ function Gradient( {
 }
 
 export default Gradient;
+export { getGradientBaseColors };
```

Comment:
I can not find the usage of that export. Do we really need that? If yes, maybe we can just do `export function getGradientBaseColors...`

Reply:
Leftover after clean up, thanks!